microsetta_interface/implementation.py

```diff
@@ -1090,6 +1098,11 @@ def get_source(*, account_id=None, source_id=None):
     per_source_not_taken = [translate_survey_template(s)
                             for s in per_source_not_taken]
 
+    # quick hack to move MyFoodRepo to top of list, if available
+    per_source_not_taken = sorted(per_source_not_taken,
+                                  key=lambda s: s['survey_template_id'],
+                                  reverse=True)
```

Comment:
```suggestion
    per_source_not_taken = per_source_not_taken[::-1]
```

As an alternative, ill defer to you which you'd prefer